commcare_connect/templates/opportunity/visit_verification.html

```diff
@@ -135,10 +135,7 @@ <h2 class="mb-2">Images</h2>
     </dl>
     {% endif %}
     <div id="action-buttons" style="margin-bottom: 25px;">
-      {% if request.org_membership.is_viewer %}
```

Comment:
why the change here around viewers?

Reply:
Fixed this. https://github.com/dimagi/commcare-connect/pull/383/commits/4c03128318ea563e84e8d138ecbac9303044870d